app/api/views/briefs.py

```diff
@@ -392,7 +393,8 @@ def get_brief(brief_id):
                    has_signed_current_agreement=has_signed_current_agreement,
                    last_edited_at=last_edited_at,
                    only_sellers_edited=only_sellers_edited,
-                   domains=domains)
+                   domains=domains,
+                   lockout_period=brief_lockout_validator.get_lockout_dates(formatted=True))
```

Comment:
```suggestion
                   lockout_period=brief_business.get_lockout_dates(formatted=True))
```

Reply:
Thanks for finding this. I'll also remove the `brief_lockout_validator` as an imported module for this file.